README.md

```diff
@@ -5,13 +5,73 @@ This repository is for EDGI [Web Monitoring Project](https://github.com/edgi-gov
 
 EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently, there is a lot of manual labor that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
 
+- [Technologies Used](#hammer-technologies-used)
 - [Project Goals](#project-goals)
 - [How to Help](#how-to-help)
 - [Project Overview](#project-overview)
 - [License & Copyright](#license--copyright)
 
 You can **track upcoming releases** by exploring our [milestones](https://github.com/edgi-govdata-archiving/web-monitoring/milestones) within the GitHub issue queue.
 
+## :hammer: Technologies Used
```

Comment:
Happy to riff on the section name of "Technologies Used"

- Technologies
- Technologies We Like (personal fave)
- Technologies Found Here
- Technologies in Residence
- Adjacent Technologies

Reply:
I appreciate the sentiment of "Technologies We Like", but I think we should avoid it here because this is in fact a list of the technologies we _use_, and it is not always the same list. :- D However, a statement mentioning our gratitude toward the open-source communities and/or companies that maintain these projects would be appropriate.